### ISSUE TITLE
Missing bn values in pre-trained state_dict

### ISSUE THREAD

**Issue by EMCarrami** [bowang-lab/scGPT#42]
Hey, 

Thanks a lot for sharing your great work :)

I noticed that the whole_human pre-trained state_dict doesn't seem to contain the trained weights for the self.bn or the self.dsbn layers. I just wanted to check if this may be due to an update to the model, and if you recommend removing these layers for inference?

Thanks.

**Comment by subercui** [comment]
Yes, we updated it during the pretraining on 33 million cells. On the other hand, we now observed little difference with or without batch norm for the fine-tuning tasks. Note the dsbn is used by default for integration with multiple batches. For other tasks, using the batch norm is optional.

**Comment by EMCarrami** [comment]
Thanks.
